### ISSUE TITLE
Logout issue

### ISSUE THREAD

**Issue by ttrevisan18** [JestonBlu/RobinHood#70]
I updated my packages and now the logout function is throwing an error. Even if I login and logout immediately. See below. I did not load the api_logout function so am not sure where the issue is being triggered. 

![image](https://user-images.githubusercontent.com/45466366/66001048-af4b6480-e46e-11e9-833f-0e3d7744511a.png)

**Comment by JestonBlu** [comment]
Thanks, ill check it out tonight.

**Comment by ttrevisan18** [comment]
I think it has to do with a conflict with api_endpoints function. When i added the "RobinHood:::" to the beginning, it seemed to fix the problem:
logout = function(RH){  
      detail <- paste("client_id=", RH$api_client_id, "&token=", 
                      RH$tokens.refresh_token, sep = "")
      url <- paste(RobinHood:::api_endpoints("revoke_token"), detail, sep = "")
      dta <- POST(url) %>% content(type = "json")
      logout <- RH$content
    }

**Comment by JestonBlu** [comment]
Thanks for the tip, ill validate and commit a fix tonight

**Comment by JestonBlu** [comment]
I have pushed a fix to master on github... there were actually a few problems... the api_logout url was missing the ? before the list of parameters in the api call. Also I standardized a lot of the names in the api functions recently and I apparently forgot to test the api_logout function. 

I'll leave this issue open for the week to see if anyone reports any other issues then Ill push the update to CRAN this weekend. Thanks for bringing it to my attention.

ead748f
